tests/unit/src/test/scala/tests/codeactions/ExtractValueLspSuite.scala

```diff
@@ -144,15 +144,15 @@ class ExtractValueLspSuite
     s"""|${ExtractValueCodeAction.title("1 + 2")}
         |${ExtractValueCodeAction.title("1 + method2(1 + 2)")}
         |${ConvertToNamedArguments.title("method2(...)")}""".stripMargin,
-    """|object Main {
-       |  def method2(i: Int) : Int = ???
-       |  def method1(s: String): Unit = {
-       |    println("Hello!"); 
-       |    val newValue = 1 + 2
-       |    method2(1 + method2(newValue))
-       |  }
-       |}
-       |""".stripMargin,
+    s"""|object Main {
+        |  def method2(i: Int) : Int = ???
+        |  def method1(s: String): Unit = {
+        |    println("Hello!");${' '}
```

Comment:
Why is this change needed? Do we even need to change this file?

Reply:
Trailing whitespaces are not the best things to have in your code as they break navigation. I spent some time on this test because my editor (not IntelliJ), removed it, and it turned out to be important in this test. In my opinion such whitespaces should be explicitly written as important. Maybe the execution of this could be better, lets say by using a space assigned to val `$whitespace`

Reply:
Sure, makes sense.